guides/v2.3/graphql/functional-testing.md

```diff
@@ -336,3 +336,113 @@ Use the following functions to cover expected exceptions:
 - `expectExceptionMessage`
 - `expectExceptionMessageRegExp`
 - `expectExceptionObject`
+
+## Run functional tests
+
+### Configure your instance
+
+1. Copy `phpunit_graphql.xml.dist` file to `phpunit_graphql.xml` in the `dev/tests/api-functional/phpunit.xml` folder.
```

Comment:
Please, change `dev/tests/api-functional/phpunit.xml` -> `dev/tests/api-functional/` if we are talking about a directory ;)

Reply:
**fixed**

Could you please review?

Thank you!